core/domain/services/admin/events/editor/AdvancedEditorData.php

```diff
@@ -180,6 +206,38 @@ protected function getEventGraphQLData($eventId)
         return compact('datetimes', 'tickets', 'prices', 'priceTypes', 'relations');
     }
 
+    /**
+     * @param int $eventId
+     * @throws DomainException
+     * @throws EE_Error
+     * @throws InvalidArgumentException
+     * @throws InvalidDataTypeException
+     * @throws InvalidInterfaceException
+     * @throws ModelConfigurationException
+     * @throws ReflectionException
+     * @throws UnexpectedEntityException
+     * @since $VID:$
+     */
+    protected function addDefaultEntities($eventId)
+    {
+        $default_dates = $this->datetime_model->create_new_blank_datetime();
+        if (is_array($default_dates) && isset($default_dates[0]) && $default_dates[0] instanceof EE_Datetime) {
```

Comment:
can this conditional be extracted into a variable?

Reply:
@tn3rb is the **owner** of that code 😄 
He knows how to handle my messed up PHP code.